### DISCUSSION TITLE
Make sqlfmt optional

### DISCUSSION THREAD

**Discussion by bkimjin** [AltimateAI/vscode-dbt-power-user#210] [General] [answered]
After the latest update, I'm bombarded by error messages that I can't disable telling me to install sqlfmt. Can we make this optional?

**Comment by mdesmet** [comment]
Interesting. Have you have enabled format on save in vscode?

**Comment by mdesmet** [comment, marked as answer]
What I can do is to implement this as a status bar instead, that would be a lot less obtrusive.

WDYT?

**Reply by bkimjin** [reply]
I have enabled format on save, and I think making it as a status bar would work. Thanks!
